File: tank/src/main/java/com/skx/tomike/tank/animation/activity/ShareElement2Activity.java
package com.skx.tomike.tank.animation.activity;

import android.os.Bundle;
import android.view.Window;

import androidx.appcompat.app.AppCompatActivity;

import com.skx.tomike.tank.R;

/**
 * 共享元素2
 */
public class ShareElement2Activity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        this.requestWindowFeature(Window.FEATURE_NO_TITLE);
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_share_element2);
    }
}

File: tank/src/main/java/com/skx/tomike/tank/animation/activity/ShareElement2Activity.kt
package com.skx.tomike.tank.animation.activity

import com.skx.common.base.BaseViewModel
import com.skx.common.base.SkxBaseActivity
import com.skx.common.base.TitleConfig
import com.skx.tomike.tank.R

/**
 * 共享元素2
 */
class ShareElement2Activity : SkxBaseActivity<BaseViewModel>() {

    override fun initParams() {}

    override fun getLayoutId(): Int {
        return R.layout.activity_share_element2
    }

    override fun configHeaderTitle(): TitleConfig {
        return TitleConfig.Builder().setTitleText("共享元素 - 界面2").create()
    }

    override fun initView() {}
}